fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/synchronizedsession/SynchronizedSessionRunner.java

```diff
@@ -206,52 +205,18 @@ public FDBDatabase getDatabase() {
     }
 
     @Override
-    public Executor getExecutor() {
-        return underlying.getExecutor();
-    }
-
-    @Override
-    @Nullable
-    public FDBStoreTimer getTimer() {
-        return underlying.getTimer();
-    }
-
-    @Override
-    public void setTimer(@Nullable FDBStoreTimer timer) {
-        underlying.setTimer(timer);
-    }
-
-    @Override
-    @Nullable
-    public Map<String, String> getMdcContext() {
-        return underlying.getMdcContext();
+    public FDBRecordContextConfig.Builder getContextConfigBuilder() {
+        return underlying.getContextConfigBuilder();
     }
 
     @Override
-    public void setMdcContext(@Nullable Map<String, String> mdcContext) {
-        underlying.setMdcContext(mdcContext);
+    public void setContextConfigBuilder(final FDBRecordContextConfig.Builder contextConfigBuilder) {
+        underlying.setContextConfigBuilder(contextConfigBuilder);
     }
 
     @Override
-    @Nullable
-    public FDBDatabase.WeakReadSemantics getWeakReadSemantics() {
-        return underlying.getWeakReadSemantics();
-    }
-
-    @Override
-    public void setWeakReadSemantics(@Nullable FDBDatabase.WeakReadSemantics weakReadSemantics) {
-        underlying.setWeakReadSemantics(weakReadSemantics);
-    }
-
-    @Nonnull
-    @Override
-    public FDBTransactionPriority getPriority() {
-        return underlying.getPriority();
```

Comment:
Looks like get/set transaction timeout millis can be similarly removed

Reply:
Removed